src/meta/model_v2/src/object.rs

```diff
@@ -66,6 +66,8 @@ pub struct Model {
     pub database_id: Option<DatabaseId>,
     pub initialized_at: DateTime,
     pub created_at: DateTime,
+    pub initialized_at_cluster_version: Option<String>,
```

Comment:
We have to add these columns for object table as well in src/meta/model_v2/migration/src/m20230908_072257_init.rs.

Reply:
Fixed.